libinteract/libinteract.py

```diff
@@ -866,6 +867,166 @@ def calc_cg_fullmatrix(identifiers, idxs, percmat, perco):
     # return the full matrix (square matrix)
     return fullmatrix
 
+def find_pos(table, name = True):
+    """Takes in a table. If name is true, returns the index of the columns
+    that have the chain name. If name is false, returns the index of the
+    columns that have the res id.
+    """
+    if name:
+        # Search for SYSTEM or any chain name between A and Z
+        positions = [i for i, elem in enumerate(table[0]) \
+                     if re.search("^[A-Z]$|^SYSTEM$", elem)]
+    else:
+        # Search for all only integers (no decimals)
+        positions = [i for i, elem in enumerate(table[0]) \
+                     if re.search("^\d+$", elem)]
+    # Make sure only two positions exist
+    assert len(positions) == 2
```

Comment:
You should raise an exception instead of using assertions (which can be disabled by using the Python interpreter in certain modes).